### PR TITLE
Add borgbackup v0.26.1

### PR DESCRIPTION
**Pull request by rmoriz**
https://github.com/borgbackup/borg

### CODE REVIEW

File: Casks/borgbackup.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  sha256 'f02231cadada5065e0cfae0ec989da8f26ba645ddf483d7ab903ddae36340432'
+
+  url "https://github.com/borgbackup/borg/releases/download/#{version}/borg-macosx64"
+  name 'BorgBackup'
+  homepage 'http://borgbackup.readthedocs.org/en/stable/'
+  license :bsd
+
+  gpg "#{url}.asc", :key_id => '243ACFA951F78E01'
+
+  container :type => :naked
```
Comment (vitorgalvao) [inline Casks/borgbackup.rb]: Are you sure this is needed?

File: Casks/borgbackup.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+cask :v1 => 'borgbackup' do
+  version '0.26.1'
+  sha256 'f02231cadada5065e0cfae0ec989da8f26ba645ddf483d7ab903ddae36340432'
+
+  url "https://github.com/borgbackup/borg/releases/download/#{version}/borg-macosx64"
+  name 'BorgBackup'
+  homepage 'http://borgbackup.readthedocs.org/en/stable/'
+  license :bsd
+
+  gpg "#{url}.asc", :key_id => '243ACFA951F78E01'
```
Comment (vitorgalvao) [inline Casks/borgbackup.rb]: This should be under `url`.

File: Casks/borgbackup.rb (hunk 1 above)
Comment (rmoriz) [inline Casks/borgbackup.rb]: It's just the binary without any container (tar, dmg) and `naked` is not auto-detected https://github.com/caskroom/homebrew-cask/blob/master/lib/hbc/container.rb

File: Casks/borgbackup.rb (hunk 2 above)
Comment (rmoriz) [inline Casks/borgbackup.rb]: fixed

**Comment by anarcat** [comment]
this is outdated: the latest borg release is 0.28.1.

**Comment by rmoriz** [comment]
The latest official binary release is still 0.26.1

File: Casks/borgbackup.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+cask :v1 => 'borgbackup' do
+  version '0.26.1'
+  sha256 'f02231cadada5065e0cfae0ec989da8f26ba645ddf483d7ab903ddae36340432'
+
+  url "https://github.com/borgbackup/borg/releases/download/#{version}/borg-macosx64"
+  gpg "#{url}.asc", :key_id => '243ACFA951F78E01'
+
```
Comment (vitorgalvao) [inline Casks/borgbackup.rb]: This blank line shouldn’t be here.

**Comment by anarcat** [comment]
@rmoriz i asked the main dev for an update - @ThomasWaldmann: where are the binaries for 0.28.1?

File: Casks/borgbackup.rb (hunk 3 above)
Comment (rmoriz) [inline Casks/borgbackup.rb]: I used the gpg-example from the docs which references https://github.com/caskroom/homebrew-cask/blob/master/Casks/libreoffice.rb including the blank line between the gpg and name part?

**Comment by anarcat** [comment]
@rmoriz in the meantime, there are binaries for 0.27...

**Comment by adityadalal924** [comment]
Note for maintainers: This is on hold until upstream binaries for v0.28.1 are created and the Cask updated accordingly.

Alternatively, if @rmoriz wants to update the cask to v0.27, this PR can be merged (and a new one opened when binaries for the latest version become available)

**Comment by anarcat** [comment]
0.28.2 was published with binaries.

File: Casks/borgbackup.rb (hunk 3 above)
Comment (adityadalal924) [inline Casks/borgbackup.rb]: How did you get this `:key_id`? (I'm not familiar with GPG)

It seems like `gpg --verify borg-macosx64.asc borg-macosx64` results in:
gpg: Signature made Sun Nov 15 18:08:48 2015 EST using RSA key ID 51F78E01 

(which is just the second half)

**Comment by adityadalal924** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 47b865e7e4d8a2f5aedec2d9254788eb956196a5. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by raxod502** [comment]
Given that Borg is an open-source command-line tool, why is it in Cask instead of Homebrew proper?

**Comment by commitay** [comment]
@raxod502 Please don't post comments on PRs that are over two years old.

If this can be added to `homebrew/core` submit a PR and it will be removed from cask.

**Pull request closed without merging**